CRITICAL - 2020-11-30 20:38:30 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:43:27 --> Undefined variable: meusDados
#0 C:\xampp\htdocs\projetoacc\app\Views\usuarios\index.php(47): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 47, Array)
#1 C:\xampp\htdocs\projetoacc\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\projetoacc\system\Common.php(1119): CodeIgniter\View\View->render('usuarios/index', Array, true)
#3 C:\xampp\htdocs\projetoacc\app\Controllers\Usuarios.php(26): view('usuarios/index', Array)
#4 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Usuarios->index()
#5 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#6 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-30 20:43:27 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:48:26 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:52:29 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:53:01 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:53:48 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:54:06 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:54:24 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:54:26 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:54:30 --> Arquivo inválido: usuarios/form.php
#0 C:\xampp\htdocs\projetoacc\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/form.p...')
#1 C:\xampp\htdocs\projetoacc\system\Common.php(1119): CodeIgniter\View\View->render('usuarios/form', Array, true)
#2 C:\xampp\htdocs\projetoacc\app\Controllers\Usuarios.php(26): view('usuarios/form', Array)
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Usuarios->index()
#4 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#5 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-30 20:54:30 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\View\Exceptions\ViewException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\View\Exceptions\ViewException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:54:32 --> Arquivo inválido: usuarios/form.php
#0 C:\xampp\htdocs\projetoacc\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('usuarios/form.p...')
#1 C:\xampp\htdocs\projetoacc\system\Common.php(1119): CodeIgniter\View\View->render('usuarios/form', Array, true)
#2 C:\xampp\htdocs\projetoacc\app\Controllers\Usuarios.php(26): view('usuarios/form', Array)
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Usuarios->index()
#4 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#5 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-30 20:54:32 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\View\Exceptions\ViewException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\View\Exceptions\ViewException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 20:56:00 --> syntax error, unexpected ';', expecting ')'
#0 C:\xampp\htdocs\projetoacc\system\Common.php(1119): CodeIgniter\View\View->render('usuarios/form', Array, true)
#1 C:\xampp\htdocs\projetoacc\app\Controllers\Usuarios.php(26): view('usuarios/form', Array)
#2 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Usuarios->index()
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#4 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-11-30 20:56:00 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ParseError), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 21:26:01 --> Undefined property: App\Controllers\Categorias::$CategoriaModel
#0 C:\xampp\htdocs\projetoacc\app\Controllers\Categorias.php(53): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 53, Array)
#1 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Categorias->store()
#2 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-30 21:26:01 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 21:40:30 --> Undefined property: App\Controllers\Categorias::$CategoriaModel
#0 C:\xampp\htdocs\projetoacc\app\Controllers\Categorias.php(53): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 53, Array)
#1 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Categorias->store()
#2 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-30 21:40:30 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 21:42:18 --> Undefined property: App\Controllers\Categorias::$CategoriaModel
#0 C:\xampp\htdocs\projetoacc\app\Controllers\Categorias.php(53): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 53, Array)
#1 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Categorias->store()
#2 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-30 21:42:18 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 21:43:25 --> Undefined property: App\Controllers\Categorias::$CategoriaModel
#0 C:\xampp\htdocs\projetoacc\app\Controllers\Categorias.php(53): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 53, Array)
#1 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Categorias->store()
#2 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-30 21:43:25 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 21:44:11 --> Undefined property: App\Controllers\Categorias::$CategoriaModel
#0 C:\xampp\htdocs\projetoacc\app\Controllers\Categorias.php(53): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 53, Array)
#1 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Categorias->store()
#2 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-30 21:44:11 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-11-30 21:44:30 --> Undefined property: App\Controllers\Categorias::$CategoriaModel
#0 C:\xampp\htdocs\projetoacc\app\Controllers\Categorias.php(53): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 53, Array)
#1 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(918): App\Controllers\Categorias->store()
#2 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#3 C:\xampp\htdocs\projetoacc\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\projetoacc\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\projetoacc\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-30 21:44:30 --> Uncaught ErrorException: Undefined variable: viewFile in C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php:308
Stack trace:
#0 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 308, Array)
#1 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(308): include()
#2 C:\xampp\htdocs\projetoacc\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
